Sourcery/SourceryTemplate.swift

```diff
@@ -44,3 +64,24 @@ private func stringContentFilter(_ name: String, _ filter: @escaping (String) ->
         return filter(s)(arg)
     }
 }
+
+private struct Filter<T> {
```

Comment:
they use extra arguments, where this just filters, we can when we add more filters but out of scope of this pr